.github/workflows/oclif-release.yml

```diff
@@ -69,6 +69,24 @@ jobs:
           INPUT_WAITING_INTERVAL: 10
           INPUT_INPUTS: '{ "formula": "${{github.event.inputs.formula}}", "version": "${{github.event.inputs.tag-name}}", "sha": "${{needs.oclif-release.outputs.sha256}}", "pre-release": "${{github.event.inputs.pre-release}}" }'
           INPUT_PROPAGATE_FAILURE: true
+  scoop-release:
+    name: Trigger scoop release workflow
+    runs-on: ubuntu-latest
+    needs: [ oclif-release ]
+    steps:
+      - name: Checkout cli repo
+        uses: actions/checkout@v2
+      - name: Invoke Scoop workflow
+        run: source .github/scripts/trigger-and-wait.sh
+        env:
+          INPUT_OWNER: twilio
+          INPUT_REPO: scoop-twilio-cli
+          INPUT_GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          INPUT_WORKFLOW_FILE_NAME: update-release.yml
+          INPUT_REF: ${{ github.event.inputs.home-brew-branch }}
```

Comment:
Why we need `home-brew-branch` branch here?

Reply:
It's the input to this workflow, for what `branch` to trigger workflow on. 
Didn't rename it throughout the file to avoid breakage with release tomorrow

Reply:
should we add a separate flag for scoop-twilio-cli branch.  btw seeing the code [here](https://github.com/twilio/twilio-cli/blob/main/.github/scripts/trigger-and-wait.sh#L75-L78) it should pick up the `main` branch by default. So we can remove INPUT_REF if we want to avoid confusion, and later on add support for new branch if there is any requirement for such in scoop-twilio-cli, @ravali-rimmalapudi thought?

Reply:
Yeah, the default value for INPUT_REF is `main`,  but when we want to do releases on different branch we need to specify this. `home-brew-branch` is related to homebrew repo. In future if there is requirement to run scoop release on different branch as @kridai suggested it is better to add separate flag `scoop-twilio-cli`.

Reply:
Makes sense!!

I've removed it for now, we can add a separate flag later on if needed!